library/core/src/intrinsics/simd.rs

```diff
@@ -2,6 +2,7 @@
 //!
 //! In this module, a "vector" is any `repr(simd)` type.
 
+#[cfg(bootstrap)]
 extern "platform-intrinsic" {
```

Comment:
could wrap the entire thing in a macro call (and `rustfmt::skip`), so there's no big diff, but just two cfg gated macro invocations.

Reply:
Fair, I gave it a shot and it wasn't that hard.